### PR TITLE
Add github watching query

### PR DESCRIPTION
**Pull request by TristanCacqueray**

### CODE REVIEW

File: haskell/src/Lentille/GitHub/Watching.hs [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+module Lentille.GitHub.Watching where
+
+import Data.Morpheus.Client qualified
+import Lentille qualified
+import Lentille.GitHub.RateLimit qualified
+import Lentille.GraphQL qualified
+import Monocle.Logging qualified
+import Monocle.Prelude
+import Monocle.Protob.Crawler qualified
```
Comment (morucci) [inline haskell/src/Lentille/GitHub/Watching.hs:16]: Fully qualified <3

**Review by morucci — APPROVED** [review]

**Comment by morucci** [comment]
Perhaps add a changelog entry to tell that the CLI get a new command.

File: haskell/src/Lentille/GitHub/Watching.hs (hunk 1 above)
Comment (TristanCacqueray) [reply haskell/src/Lentille/GitHub/Watching.hs:16]: I think that's the way to go :)

**Review by TristanCacqueray — COMMENTED** [review]
I meant to add a new crawler to add all the projects watched by a given user. But perhaps that could be done in a followup PR?

**Pull request merged**